### DISCUSSION TITLE
Перестали синхронизироваться расходы

### DISCUSSION THREAD

**Discussion by lewadedun** [ardov/zerro#132] [General]
Около недели как перестали синхронизироваться операции... У меня одного так?

**Comment by ardov** [comment]
@lewadedun, а на zenmoney.ru всё есть? Если там нет, значит приложение ДМ по какой-то причине не отправляет данные на сервер.

**Reply by lewadedun** [reply]
Там все есть

**Reply by ardov** [reply]
Хм, тогда можно попробовать в инкогнито зайти в зерро. Если там всё есть, то можно разлогиниться и заново зайти.
Ещё интересно в консоли посмотреть что за ошибка при запросе данных, если получится, это помогло бы.

На всякий случай, как посмотреть ошибку:
- Надо в браузере открыть консоль Ctrl+Alt+I или Cmd+Opt+I, переключиться на вкладку Network или Сеть.
- Запустить синхронизацию в Зерро
- Строчка запроса появится в консоли, надо на неё кликнуть и посмотреть что отдаёт Дзенмани в ответе

**Comment by lewadedun** [comment]
огда можно попробовать в инкогнито зайти в зерро - сработало ))))

пт, 9 июн. 2023 г. в 05:45, Alexey Ardov ***@***.***>:

> Хм, тогда можно попробовать в инкогнито зайти в зерро. Если там всё есть,
> то можно разлогиниться и заново зайти.
> Ещё интересно в консоли посмотреть что за ошибка при запросе данных, если
> получится, это помогло бы.
>
> На всякий случай, как посмотреть ошибку:
>
>    - Надо в браузере открыть консоль Ctrl+Alt+I или Cmd+Opt+I,
>    переключиться на вкладку Network или Сеть.
>    - Запустить синхронизацию в Зерро
>    - Строчка запроса появится в консоли, надо на неё кликнуть и
>    посмотреть что отдаёт Дзенмани в ответе
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ardov/zerro/discussions/132#discussioncomment-6127731>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACHSUD4LP5C6ANEEKGE6WXTXKJWZNANCNFSM6AAAAAAY7AGPN4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
С уважением,
Дедун Алексей Александрович
